Tweetbook.Contracts/V1/Requests/Queries/PaginationQuery.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Tweetbook.Contracts.V1.Requests.Queries
+{
+    public class PaginationQuery
+    {
+        public PaginationQuery()
+        {
+            PageNumer = 1;
+            PageSize = 100;
+        }
+
+        public PaginationQuery(int pageNumber, int pageSize)
+        {
+            PageNumer = pageNumber;
+            PageSize = pageSize;
+        }
+
+        public int PageNumer { get; set; }
+
+        public int PageSize { get; set; }
```

Comment:
You can move these at the beginning, because we tend to put upfront what is easily readible and doesn't need much attention